content/guides/examples/websites.md

```diff
@@ -119,13 +119,16 @@ you plan to use, you can find it using the command like:
 ```bash
 $ nslookup gateway.ipfs.io
 ```
-And noting the IPv4 addresses returned. You should create an A record for each address.
+1. Note the IP addresses returned.
+1. Create an A record for each IPv4 address (all results that look like 192.0.2.71).
+1. Create an AAAA record for each IPv6 address (all results that look like 2001:DB8:324A::A1).
+1. Periodically check if one of these addresses changed and update the record.
```

Comment:
```suggestion
Note: The ipfs.io gateway IP addresses wont change, so you can set them and forget them. If you use a custom gateway where you don't control the IP address and they could change you may need to re-check them periodically and update your DNS records if they do.
```

Reply:
Resolved in in https://github.com/ipfs/docs/commit/95275d753f17d5555c2b662260af1cf08061c63f